[ERROR] 2018-10-24 02:51:52,239 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 02:52:52,311 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 02:53:52,382 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 02:54:52,452 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 02:55:52,523 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 02:56:52,594 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 02:57:52,663 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 02:58:52,733 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 02:59:52,803 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:00:52,873 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:01:52,944 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:02:53,014 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:03:53,085 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:04:53,154 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:05:53,225 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:06:53,295 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:07:53,366 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:08:53,437 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:09:53,507 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:10:53,577 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:11:53,646 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:12:53,714 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:13:53,784 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:14:53,852 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:15:53,920 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:16:53,990 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:17:54,057 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:18:54,128 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:19:54,198 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:20:54,268 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:21:54,335 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:22:54,406 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:23:54,476 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:24:54,545 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:25:54,613 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:26:54,683 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:27:54,753 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:28:54,823 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:29:54,894 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:30:54,964 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:31:55,035 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:32:55,105 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:33:55,174 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:34:55,244 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:35:55,313 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:36:55,382 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:37:55,451 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:38:55,521 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:39:55,591 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:40:55,661 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:41:55,731 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:42:55,801 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:43:55,871 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:44:55,942 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:45:56,011 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:46:56,081 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:47:56,149 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:48:56,219 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:49:56,289 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:50:56,359 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:51:56,429 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:52:56,499 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:53:56,569 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:54:56,639 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:55:56,709 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:56:56,779 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:57:56,849 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:58:56,919 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 03:59:56,989 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 04:00:57,060 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 04:01:57,131 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 04:02:57,201 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 04:03:57,272 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 04:04:57,343 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 04:05:57,414 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 04:06:57,485 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 04:07:57,555 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 04:08:57,626 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 04:09:57,696 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 04:10:57,767 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 04:11:57,838 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 04:12:57,908 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 04:13:57,978 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

